lib/bin/lwm2m_carrier/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,28 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+liblwm2m_carrier 0.20.0
+***********************
+
+Release for modem firmware version 1.3.0.
+
+Certification status
+====================
+
+For certification status, see `Mobile network operator certifications`_.
+
+Changes
+=======
+
+* CA certificates are no longer provided by the LwM2M carrier library.
+
+  * Application is now expected to store CA certificates into the modem security tags.
+  * Added a new event :c:macro:`LWM2M_CARRIER_EVENT_CERTS_INIT` that instructs the application to provide the CA certificate security tags to the LwM2M carrier library.
+* Renamed the event :c:macro:`LWM2M_CARRIER_BSDLIB_INIT` to :c:macro:`LWM2M_CARRIER_EVENT_MODEM_INIT`.
+* Added a new error code :c:macro:`LWM2M_CARRIER_ERROR_SERVICE_UNAVAILABLE` that indicates that the LwM2M server is unavailable due to maintenance.
```

Comment:
I believe `LWM2M_CARRIER_ERROR_CONFIGURATION` was also added recently, should it be mentioned here?